src/logsearch-filters-common/target/logstash-filters-default.conf.erb

```diff
@@ -0,0 +1,128 @@
+if [@type] in ["syslog", "relp"] {
```

Comment:
After agreeing that this should be moved into the logstash package, I actually think that your final argument `we might have a need to make something here user-configurable` is actually the clincher as to why it should remain in the job.  (which in turn requires commiting 2 copies of the compiled file; one which "normal", and one which is "escaped" so it can be passed through BOSH's template generating ERB process)